code/Model/Submission/SubmittedFormField.php

```diff
@@ -75,14 +75,12 @@ public function canDelete($member = null)
 
     /**
      * Generate a formatted value for the reports and email notifications.
-     * Converts new lines (which are stored in the database text field) as
-     * <brs> so they will output as newlines in the reports
      *
      * @return string
      */
     public function getFormattedValue()
     {
-        return nl2br($this->dbObject('Value')->ATT());
+        return $this->dbObject('Value')->RAW();
```

Comment:
I agree and disagree with this change.

Agree: adding line breaks and things "so they will output as newlines in the reports" should be done in the report

Disagree: the method is called "getFormattedValue", but it no longer returns a formatted value

Reply:
Yep, agreed. Have renamed to better indicate its functionality.

Reply:
Unfortunately we can't rename the function without breaking semver, but you could add a new function and use the new one while leaving the old one in place